nbsite/pyodide/__init__.py

```diff
@@ -1,8 +1,8 @@
+import fcntl
```

Comment:
This is not available on Windows

Reply:
Indeed, I don't think there's a safe way to lock a file on windows at all.